testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,202 @@ QUnit.module('Form', () => {
+                                    editorType,
+                                }, ],
+                            },
+                        ],
+                    },
+
+                ],
+            });
+            const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+            assert.strictEqual($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS), false);
+        });
+
+        ['left', 'right'].forEach(labelLocation => {
+            test(`${editorType} should not have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when the labelLocation=${labelLocation}`, function(assert) {
+                const $form = $('#form').dxForm({
+                    labelLocation,
+                    items: [
+                        {
+                            itemType: 'group',
+                            items: [
+                                {
+                                    itemType: 'group',
+                                    items: [{
+                                        dataField: editorType,
+                                        label: { visible: true },
+                                        editorType,
+                                    }, ],
+                                },
+                            ],
+                        },
+
+                    ],
+                });
+                const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+                assert.strictEqual($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS), false);
+            });
+        });
+
+        ['center', 'right'].forEach(alignment => {
+            test(`${editorType} should not have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when label.alignment=${alignment}`, function(assert) {
+                const $form = $('#form').dxForm({
+                    labelLocation: 'top',
+                    items: [
+                        {
+                            itemType: 'group',
+                            items: [
+                                {
+                                    itemType: 'group',
+                                    items: [{
+                                        dataField: editorType,
+                                        label: { visible: true, alignment },
+                                        editorType,
+                                    }, ],
+                                },
+                            ],
+                        },
+
+                    ],
+                });
+                const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+                assert.strictEqual($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS), false);
+            });
+        });
+
+        test(`default editor should not have a css class ${TOGGLE_CONTROLS_PADDING_CLASS} when label.alignment=left, label.visible=true,  labelLocation=top`, function(assert) {
```

Comment:
It does not matter which value `label.alignment`, `label.visible` or `labelLocation` has for this test. This class should not be rendered if editorType is not CheckBox, Switch or RadioGroup.
```suggestion
        test('editor should not have class if it is not CheckBox, Switch or RadioGroup', function(assert) {
```

Reply:
Thanks